Sources/WebDriver/Session.swift

```diff
@@ -321,6 +328,14 @@ public class Session {
         try webDriver.send(Requests.SessionWindowSize.Post(session: id, windowHandle: handle, width: width, height: height))
     }
 
+    /// - Parameters:
+    ///   - latitude: Number coordinate of current geo location
+    ///   - longitude: Number coordinate of current geo location
+    ///   - altitude: Number coordinate of current geo location
+    public func setLocation(latitude: Double, longitude: Double, altitude: Float) throws {
```

Comment:
This should take a `Location` type as a parameter, or at least have an overload that takes it.